### DISCUSSION TITLE
Adding URL in inspector

### DISCUSSION THREAD

**Discussion by RajeshNakkanaboina** [clientIO/joint#2267] [Q&A]
### Introduction

Hi Team,

In the inspector container for the inscribed circular shape, I want to add a URL field just beneath the text field. From there, the user must go to that URL page.
![Screenshot (117)](https://github.com/clientIO/joint/assets/134581112/c4bbc99a-69ef-4a04-9073-e1e4c6975d38)
Check the above screenshot for reference 

Thanks in advance

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment]
> From there, the user must go to that URL page.

Could you elaborate, please?
